### DISCUSSION TITLE
请教一下：云盘Api代理和数据传输代理具体的含义是什么呢？

### DISCUSSION THREAD

**Discussion by linwhth** [cloud-fs/cloud-fs.github.io#77] [Ideas]
希望能具体讲解一下云盘Api代理和数据传输代理。
使用pikpak的话，有必要云盘Api代理和数据传输代理全部设置吗？

**Comment by cloud-fs** [comment]
文件上传下载走的数据传输代理，其它请求都是api代理，根据你自己的情况设置
